backend/app/api/endpoints/adapter/subscription_follows.py

```
@@ -119,6 +134,54 @@ def get_followers_count(
+    try:
+        return subscription_notification_service.update_follow_settings(
+            db=db,
+            subscription_id=subscription_id,
+            user_id=current_user.id,
+            notification_level=request.notification_level,
+            notification_channel_ids=request.notification_channel_ids,
+        )
+    except ValueError as e:
+        from fastapi import HTTPException
+
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)